desktop/apps/auction_support/client/registration_form.coffee

```diff
@@ -103,18 +110,19 @@ module.exports = class RegistrationForm extends ErrorHandlingForm
       else
         resolve()
 
+  # Lock the form- action.finally() callback executes when form submission is complete regardless of success
   loadingLock: ($element, action) ->
     return if $element.hasClass('is-loading')
     $element.addClass 'is-loading'
-    action().finally => $element.removeClass 'is-loading'
+    action().finally => $element.removeClass 'is-loading' unless @comboForm
```

Comment:
I was moving on but i found the reason. in `ErrorHandlingForm@showError`:
   ```coffeescript
@$submit.removeClass('is-loading').before "<div class='error'>#{message}</div>"
    analyticsHooks.trigger 'error', description
```
This does rely on a @$submit element being defined, but it is.